src/litserve/server.py

```diff
@@ -54,7 +56,9 @@ def load_and_raise(response):
         pickle.loads(response)
         raise HTTPException(500, "Internal Server Error")
     except pickle.PickleError:
-        logging.error(f"Expected response to be a pickled exception, but received an unexpected response: {response}.")
+        logger.exception(
+            f"Expected response to be a pickled exception, but received an unexpected response: {response}."
```

Comment:
It's quite hard to understand this message if you don't have the full picture. Can we make it easier to parse? Like: Error cannot be communicated from the worker to the main process because it failed to be pickled etc.